source/mongodump.txt

```diff
@@ -264,6 +264,13 @@ Behavior
    the :dbtools:`--archive </mongodump/#std-option-mongodump.--archive>` 
    option.  
 
+Using ``mongodump`` Without an ``authSource``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When :urioption:`authSource` is not specified in the MongoDB URI, the database name provided 
+with :option:`--db` is used to authenticate your ``mongodump`` session
+and **not** indicate the database being dumped.
```

Comment:
[question]

This might just be my unfamiliarity with the tool, but in this scenario where is the db to dump specified?

Reply:
I tested this and posed the question in #mongo-tools, the answer is that if you do not provide authSource, the `--db` database is used as BOTH the authenticating database and the database being dumped. This is slightly counter to the ticket description, but I have updated this note to reflect.